blazor/rich-text-editor/video.md

```diff
@@ -9,24 +9,23 @@ documentation: ug
 
-The Rich Text Editor allows you to insert videos from online sources and local computers where you want to insert the video in your content. For inserting the video to the Rich Text Editor, the following list of options have been provided in the [RichTextEditorVideoSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorVideoSettings.html).
+The Rich Text Editor allows you to insert videos from online sources and local computers where you want to insert the video in your content. For inserting the video into the Rich Text Editor, the following list of options has been provided in the [RichTextEditorVideoSettings](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorVideoSettings.html).
 
 | Options | Description |
 |----------------|---------|
-| AllowedTypes | Specifies the extensions of the video types allowed to insert on bowering and passing the extensions with comma separators. For example, pass allowedTypes as `.mp4`, `.mov`, `.wmv` and `.avi`.|
-| LayoutOption | Sets the default display for an video when it is inserted in to the Rich Text Editor. Possible options are: `Inline` and `Break`.|
-| SaveFormat | Sets the default save format of the video element when inserted. Possible options are: `Blob` and `Base64`.|
-| Width | Sets the default width of the video when it is inserted in the Rich Text Editor.|
-| MinWidth | Sets the minWidth of the video element when it is inserted in the Rich Text Editor.|
-| MaxWidth | Sets the maxWidth of the video element when it is inserted in the Rich Text Editor.|
-| Height | Sets the default height of the video when it is inserted in the Rich Text Editor.|
-| MinHeight | Sets the minHeight of the video element when it is inserted in the Rich Text Editor.|
-| MaxHeight | Sets the maxHeight of the video element when it is inserted in the Rich Text Editor.|
-| SaveUrl | Provides URL to map the action result method to save the video.|
-| RemoveUrl | Provides URL to map the action result method to remove the video.|
-| Path | Specifies the location to store the video.|
-| Resize | Sets the resizing action for the video element.|
-| ResizeByPercent | Sets the percentage values for the video element with the resizing action.|
+| [AllowedTypes](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.RichTextEditorVideoSettings.html#Syncfusion_Blazor_RichTextEditor_RichTextEditorVideoSettings_AllowedTypes) | Specifies the extensions of the video types allowed to insert on bowering and passing the extensions with comma separators. For example, pass allowedTypes as `.mp4`, `.mov`, `.wmv` and `.avi`.|
+| [VideoLayoutOption](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.VideoToolbarCommand.html#Syncfusion_Blazor_RichTextEditor_VideoToolbarCommand_VideoLayoutOption) | Sets the default display for an video when it is inserted in to the Rich Text Editor. Possible options are: `Inline` and `Break`.|
```

Comment:
| [VideoLayoutOption](https://help.syncfusion.com/cr/blazor/Syncfusion.Blazor.RichTextEditor.VideoToolbarCommand.html#Syncfusion_Blazor_RichTextEditor_VideoToolbarCommand_VideoLayoutOption) | Sets the default display for a video when it is inserted into the Rich Text Editor. Possible options are `Inline` and `Break.` |